docs/advanced.md

```diff
@@ -87,6 +87,34 @@ RKE2 supports encrypting Secrets at rest, and will do the following automaticall
 
 Once enabled any created secret will be encrypted with this key. Note that if you disable encryption then any encrypted secrets will not be readable until you enable encryption again using the same key.
 
+## Single-Server Encryption Key Rotation
+
+To rotate secrets encryption keys on a single-node cluster:
+
+- Start the RKE2 server with the flag `--secrets-encryption`
```

Comment:
Unlike K3s, secrets encryption is hardcoded on in RKE2. While it can be set via the CLI, the provided value is ignored at runtime and forced to `true`.